ledgereth/__main__.py

```diff
@@ -121,13 +136,22 @@ def send_value(dongle, args):
         dongle.close()
         sys.exit(1)
 
+    if not args.gasprice and not args.max_fee:
+        print(
+            "Either --gasprice or --max-fee must be provided", file=sys.stderr
+        )
+        dongle.close()
+        sys.exit(1)
```

Comment:
sometimes people use exit code 2 for bad args stuff

Reply:
I haven't seen much of a convention in POSIX, unless there's something I missed?  But I'm cool with that change if you would find it useful for scripting.

Reply:
I usually just check if it is not `0` if bash scripting tbh, so this is not a big deal.

But I just noticed `click` uses `2` for argument errors but `argparse.ArgumentError` uses `1`; probably good to stay consistent with `argparse` for now

Reply:
ah interesting, I see ArgumentParser does exit cleanly with code `2` if you provide an invalid type value.

    (ledger-eth-lib) [mike@megan ledger-eth-lib]$ python -m ledgereth send FROM TO 100000000000000000 -n asdf -p 50100000000 -c 137
    usage: __main__.py send [-h] -n NONCE [-c CHAINID] [-g GAS] [-p GASPRICE] [-f MAX_FEE] [-b PRIORITY_FEE] [-d DATA] FROM TO WEI
    __main__.py send: error: argument -n/--nonce: invalid int value: 'asdf'
    (ledger-eth-lib) [mike@megan ledger-eth-lib]$ echo $?
    2

But throwing `ArgumentTypeError` results in an unhandled error which apparently uses code `134`.

Either way, exit code `2` seems like a good call here,